conda/cli/install.py

```diff
@@ -64,7 +67,57 @@
 stderrlog = getLogger("conda.stderr")
 
 
-def check_prefix(prefix, json=False):
+def validate_prefix_exists(prefix: str | Path) -> Path:
```

Comment:
Should we always promote to `Path`? Looks like a side effect that might be surprising to those passing a string.